src/Microsoft.AspNet.Mvc.Core/Filters/IFormatFilter.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    ///  A filter which produces a desired content type for the current request. 
+    /// </summary>
+    /// <remarks>A FormatFilter decides what content type to use if a format value is present in the URL. 
```

Comment:
Reposting because I don't see a response or a change:

> if you're referring to the class use <see...>. Also it's not clear that you're referring to an implementation, or what value this comment adds. Suggest just removing it. If there's something you think is important to note here let's discuss and come up with a better wording